### DISCUSSION TITLE
Later objects should appear over older objects in Editor's Timeline

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#24212] [Ideas]
At the moment it's the opposite of how it looks in Stable.
When it's set like it is now, I can't tell if I placed the slider directly on (1)'s slider end, or 1/4 beat later (it's actually 1/4 beat later, and it's practically not possible to see that)
![image](https://github.com/ppy/osu/assets/82095491/19a7b2e3-d2e3-4e9a-b8a0-7ee60bd55e35)

**Comment by peppy** [comment]
Even if they were flipped in the above scenario, you couldn't be able to tell in the other direction, so this sounds like irrational reasoning.

You could zoom in though? Or you could look for the stacking hint in the timeline:

![2023-07-13 23 28 46](https://github.com/ppy/osu/assets/191335/9faf9b98-41a2-4805-8b17-e1630bcf7610)

Now how it visually stacks in the vertical direction.
